com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -0,0 +1,80 @@
+---
+id: com.github.jeromerobert.pdfarranger
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+command: pdfarranger
+# These renames will be obsolete with 
+# https://github.com/jeromerobert/pdfarranger/issues/158
+# Currently known bug: Missing icon in about dialog (Will be fixed by #158.)
+rename-icon: pdfarranger
+rename-desktop-file: pdfarranger.desktop
+rename-appdata-file: pdfarranger.appdata.xml
+modules:
+- name: intltool
+  cleanup: [ "*" ]
+  sources:
+  - type: archive
+    url: https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+    sha256: 67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+- name: openjpeg # dependency of poppler
+  buildsystem: cmake
+  sources:
+  - url: https://github.com/uclouvain/openjpeg/archive/v2.3.1.tar.gz
+    sha256: 63f5a4713ecafc86de51bfad89cc07bb788e9bba24ebbf0c4ca637621aadb6a9
+    type: archive
+    config-opts: [ "-DBUILD_STATIC_LIBS=OFF" ]
+- name: poppler
+  buildsystem: cmake
+  sources:
+  - url: https://poppler.freedesktop.org/poppler-0.86.1.tar.xz
+    sha256: af630a277c8e194c31339c5446241834aed6ed3d4b4dc7080311e51c66257f6c
+    type: archive
+    config-opts: 
+    - "-DENABLE_UTILS=OFF"
+    - "-DENABLE_CPP=OFF"
+    - "-DENABLE_QT5=OFF"
```

Comment:
This never worked due to wrong indentation:
`(flatpak-builder:13071): flatpak-builder-WARNING **: 13:41:45.949: Unknown property config-opts for type BuilderSourceArchive`